SBVUnitTest/TestSuite/Basics/ProofTests.hs

```diff
@@ -9,26 +9,30 @@
 -- Test suite for Examples.Basics.ProofTests
 -----------------------------------------------------------------------------
 
-module TestSuite.Basics.ProofTests(testSuite)  where
+module TestSuite.Basics.ProofTests(tests)  where
 
 import Data.SBV
 
 import Examples.Basics.ProofTests
 import SBVTest
 
--- Test suite
-testSuite :: SBVTestSuite
-testSuite = mkTestSuite $ \_ -> test [
-   "proofs-1"  ~: assert       =<< isThm f1eqf2
- , "proofs-2"  ~: assert . not =<< isThm f1eqf3
- , "proofs-3"  ~: assert       =<< isThm f3eqf4
- , "proofs-4"  ~: assert       =<< isThm f1Single
- , "proofs-5"  ~: assert       =<< isSat (f1 `xyEq` f2)
- , "proofs-6"  ~: assert       =<< isSat (f1 `xyEq` f3)
- , "proofs-7"  ~: assert . not =<< isSat (exists "x" >>= \x -> return (x .== x + (1 :: SWord16)))
- , "proofs-8"  ~: assert       =<< isSat (exists "x" >>= \x -> return (x :: SBool))
- , "proofs-9"  ~: assert       =<< isSat (exists "x" >>= \x -> return x :: Predicate)
- ]
- where func1 `xyEq` func2 = do x <- exists_
-                               y <- exists_
-                               return $ func1 x y .== func2 x (y :: SWord8)
+tests :: TestTree
+tests =
+  testGroup "Basics.Proof"
+    [ testCase "proofs-1" (assertIsThm f1eqf2)
+    , testCase "proofs-2" (assertIsntThm f1eqf3)
+    , testCase "proofs-3" (assertIsThm f3eqf4)
+    , testCase "proofs-4" (assertIsThm f1Single)
+    , testCase "proofs-5" (assertIsSat (f1 `xyEq` f2))
+    , testCase "proofs-6" (assertIsSat (f1 `xyEq` f3))
+    , testCase "proofs-7" (assertIsntSat (exists "x" >>= \x -> return (x .== x + (1 :: SWord16))))
+    , testCase "proofs-8" (assertIsSat (exists "x" >>= \x -> return (x :: SBool)))
+    , testCase "proofs-9" (assertIsSat (exists "x" >>= \x -> return x :: Predicate))
```

Comment:
I might start lumping tests like these under one test case (e.g. "proofs"). I have moved towards that style to reduce noise, since all I care about is a general indicator that some test is failing. Once I get a failure, then I drill down.